lambdas/navigator_config/index.py

```diff
@@ -14,16 +15,40 @@ def handler(event, context):
         props = event['ResourceProperties']
         config(props['DestBucket'], props['DestDir'], props['ConfigEsUrl'],
                props['ConfigApiUrl'], props['ConfigS3Bucket'],
-               props['BucketTitle'], props['BucketIcon'], props['BucketDescription'])
+               props['BucketTitle'], props['BucketIcon'], props['BucketDescription'],
+               props['RegistryHost'])
         cfnresponse.send(event, context, cfnresponse.SUCCESS, {})
     except Exception as e:
         cfnresponse.send(event, context, cfnresponse.FAILED, {})
         raise
 
-def config(dest_bucket, dest_dir, es_url, api_url, s3_bucket,
-           bucket_title, bucket_icon, bucket_description):
-    region = boto3.session.Session().region_name
+def print_json(source):
+    """
+    Pretty-print config as JSON.
+    """
+    return json.dumps(source, sort_keys=True, indent=4, separators=(',', ': '))
+
+def update_config(existing, autogenerated):
+    """
+    Update existing config with new autogenerated config.
+
```

Comment:
Add a comment in the docstring that it updates keys based on top level in json and not nested keys. I think its fairly clear since you named it update instead of merge, but might as well be explicit